src/view/trip-info-view.js

```diff
@@ -0,0 +1,75 @@
+import dayjs from 'dayjs';
+
+import AbstractView from '../framework/view/abstract-view';
+
+import { humanizeEventDate, sortByDate } from '../utils/event';
+
+const createSummaryBoardTemplate = (events, destinations, tripPrice) => {
+  const getDestination = (id) => destinations.find((el) => el.id === id).name;
+
+  const getRoute = () => {
+    if (!events.length) {
+      return 'Add your first event!';
+    }
+
+    if (events.length > 3) {
+      const firstDestination = getDestination(events[0].destination);
+      const lastDestination = getDestination(events[events.length - 1].destination);
+
+      return `${firstDestination} — ... — ${lastDestination}`;
+    }
+
+    return events.map((el) => getDestination(el.destination)).join(' — ');
+  };
+
+  const getDates = () => {
+    if (!events.length) {
+      return '';
+    }
+
+    const firstDate = humanizeEventDate(events[0].dateFrom, true);
+    const lastDate = humanizeEventDate(events[events.length - 1].dateFrom, true);
```

Comment:
Можно воспользоваться новым синтаксисом
```suggestion
    const lastDate = humanizeEventDate(events.at(- 1).dateFrom, true);
```

Reply:
Ready